lib/cufft/fft.jl

```diff
@@ -25,32 +25,34 @@ Base.:(*)(p::ScaledPlan, x::DenseCuArray) = rmul!(p.p * x, p.scale)
 
 # N is the number of dimensions
 
-mutable struct CuFFTPlan{T<:cufftNumber,S<:cufftNumber,K,inplace,N} <: Plan{S}
+mutable struct CuFFTPlan{T<:cufftNumber,S<:cufftNumber,K,inplace,N,R,B} <: Plan{S}
     # handle to Cuda low level plan. Note that this plan sometimes has lower dimensions
     # to handle more transform cases such as individual directions
     handle::cufftHandle
     ctx::CuContext
     stream::CuStream
     input_size::NTuple{N,Int}   # Julia size of input array
     output_size::NTuple{N,Int}  # Julia size of output array
-    region::Any
+    region::NTuple{R,Int}
+    buffer::B                   # buffer for out-of-place complex-to-real FFT
```

Comment:
```suggestion
    buffer::B                   # buffer for out-of-place complex-to-real FFT, or `nothing` if not needed
```

Reply:
Feel free to squash merge without running CI.

Reply:
For information, I can't bypass the auto-merge.

Reply:
Good to know. You're already in a team with `write` access; I've bumped that to `maintainer` access. That should also make it possible for you to manage issues and PRs.